branchpro/apps/_simulation.py

```diff
@@ -77,8 +77,13 @@ def __init__(self):
                     html.H1('Branching Processes'),
```

Comment:
This is completely optional, but if you add this url to external_stylesheets in the dash app, it will draw the button in a similar style to the rest of the dash interface.
```
            __name__, external_stylesheets=[
                dbc.themes.BOOTSTRAP,
                'https://codepen.io/chriddyp/pen/bWLwgP.css'])
```